### PR TITLE
challenge 1 migration wagmi, viem, daisyUI

### PR DESCRIPTION
**Pull request by technophile-04**
Sorry for the commit messages, I by mistake pushed it directly to main challenge branch so had to revert the those changes their and revert the changes here too to go to merge commit 🙌

### CODE REVIEW

File: packages/nextjs/pages/stakings.tsx [hunk 1]
```diff
@@ -9,8 +9,9 @@ const Stakings: NextPage = () => {
   const { data: stakeEvents, isLoading } = useScaffoldEventHistory({
     contractName: "Staker",
     eventName: "Stake",
-    fromBlock: 0,
+    fromBlock: 0n,
   });
+  console.log("Stakering Events", stakeEvents);
```
Comment (rin-st) [inline packages/nextjs/pages/stakings.tsx:14]: log

File: packages/nextjs/components/stake/StakeContractInteraction.tsx [hunk 2]
```diff
@@ -87,12 +87,12 @@ export const StakeContractInteraction = ({ address }: { address?: string }) => {
         <div className="flex items-start justify-around w-full">
           <div className="flex flex-col items-center justify-center w-1/2">
             <p className="block text-xl mt-0 mb-1 font-semibold">Time Left</p>
-            <p className="m-0 p-0">{timeLeft ? `${humanizeDuration(timeLeft.toNumber() * 1000)}` : 0}</p>
+            <p className="m-0 p-0">{timeLeft ? `${humanizeDuration(parseFloat(timeLeft.toString()) * 1000)}` : 0}</p>
```
Comment (rin-st) [inline packages/nextjs/components/stake/StakeContractInteraction.tsx:90]: you can use Number(timeLeft) * 1000

File: packages/nextjs/pages/stakings.tsx (hunk 1 above)
Comment (technophile-04) [reply packages/nextjs/pages/stakings.tsx]: 🙌 removed at [72b0d2e](https://github.com/scaffold-eth/se-2-challenges/pull/78/commits/72b0d2e87f9de80a69e7fe6e39ab574ae3e8fb01)

File: packages/nextjs/components/stake/StakeContractInteraction.tsx [hunk 3]
```diff
@@ -87,7 +87,7 @@ export const StakeContractInteraction = ({ address }: { address?: string }) => {
         <div className="flex items-start justify-around w-full">
           <div className="flex flex-col items-center justify-center w-1/2">
             <p className="block text-xl mt-0 mb-1 font-semibold">Time Left</p>
-            <p className="m-0 p-0">{timeLeft ? `${humanizeDuration(parseFloat(timeLeft.toString()) * 1000)}` : 0}</p>
+            <p className="m-0 p-0">{timeLeft ? `${humanizeDuration(Number(timeLeft.toString()) * 1000)}` : 0}</p>
```
Comment (rin-st) [inline packages/nextjs/components/stake/StakeContractInteraction.tsx:90]: you don't need toString )

**Review by rin-st — APPROVED** [review]

**Comment by rin-st** [comment]
Thanks! Looks good to me!

**Review by Pabl0cks — APPROVED** [review]
GJ Shiv! It's working great for me 🙌

**Comment by technophile-04** [comment]
Tysm guys, merging this !

**Pull request merged**